src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/camunda/CamundaVariableDefinition.java

```diff
@@ -28,7 +28,8 @@
     WARNING_LIST("warningList"),
     CFT_TASK_STATE("cftTaskState"),
     WORK_TYPE("workType"),
-    CASE_MANAGEMENT_CATEGORY("caseManagementCategory");
+    CASE_MANAGEMENT_CATEGORY("caseManagementCategory"),
+    DESCRIPTION("description");
```

Comment:
Is it a Camunda variable?  This is an R2 feature so not sure it should be in Camunda.

Reply:
Would not be nice to have in Camunda when we are looking at a task in the cockpit?

Reply:
@vgulapally , this is R2 feature.

Reply:
CamundaVariableDefinition is also used in R2, CFTTaskMapper uses this